release.ps1

```diff
@@ -57,11 +61,35 @@ ce octo build-information `
   --space="${SPACE_NAME}" `
   --overwrite-mode=OverwriteExisting
 
+Write-Output "Writing Release Notes"
+$releaseNotes = ""
+$eventPayload.commits | ForEach-Object {
+  $releaseNotes += @"
+#### commit $($_.id)
+
+**Author:** $($_.author.username) - $($_.author.name) <$($_.author.email)>
+
+**Committer:** $($_.committer.username) - $($_.committer.name) <$($_.committer.email)>
+
+**Date:**   $(Get-Date $_.timestamp -Format "dddd MM/dd/yyyy HH:mm K")
+
+``````text
```

Comment:
Just being compulsive, but would adding a line break before the ` `````` ` make hide the un-rendered markdown? Will post image in PR comment

Reply:
Fixed. How about now? https://octopus.apps.ops-drivevariant.com/app#/Spaces-1/projects/demoapp/deployments/releases/0.1.0-f-vd0001.395

Reply:
perfect